app/views/providers/check_provider_answers/index.html.erb

```diff
@@ -76,7 +76,7 @@
         name: :proceeding_type,
         url: providers_legal_aid_application_proceedings_types_path(anchor: 'proceeding-search-input'.freeze),
         question: t('.section_proceeding.proceeding'),
-        answer: @proceeding_types.first.meaning,
+        answer: @proceeding_type.meaning,
```

Comment:
see comment in controller about using `@lead_proceeding_type` as the variable name

Reply:
Good spot